core/web/resolver/mutation.go

```diff
@@ -310,3 +312,45 @@ func (r *Resolver) DeleteOCRKeyBundle(ctx context.Context, args struct {
 
 	return NewDeleteOCRKeyBundlePayloadResolver(deletedKey, nil), nil
 }
+
+func (r *Resolver) CreateNode(ctx context.Context, args struct {
+	Input *types.NewNode
+}) (*CreateNodePayloadResolver, error) {
+	if err := authenticateUser(ctx); err != nil {
+		return nil, err
+	}
+
+	node, err := r.App.EVMORM().CreateNode(*args.Input)
+	if err != nil {
+		return nil, err
+	}
+
+	return NewCreateNodePayloadResolver(&node), nil
+}
+
+func (r *Resolver) DeleteNode(ctx context.Context, args struct {
+	ID int32
+}) (*DeleteNodePayloadResolver, error) {
+	if err := authenticateUser(ctx); err != nil {
+		return nil, err
+	}
+
+	node, err := r.App.EVMORM().Node(args.ID)
+	if err != nil {
+		if errors.Is(err, sql.ErrNoRows) {
+			return NewDeleteNodePayloadResolver(nil, err), nil
+		}
```

Comment:
Why do we need to do a Get node prior to just calling `DeleteNode` below and checking (as you already do) `evm.ErrNoRowsAffected`? Isn't `ErrNoRowsAffected` when deleting something the same as `ErrNoRows` when searching for something?

Reply:
Yeah, it is the same, I just wanted to make clear that the mutation is handling both possible errors, though it should only occur one of them, if it happen.

As for the fetch, each delete mutation returns a copy of the resource it is trying to delete, at least that's what I've seen so far in the GQL implementation, so since the ORM it is not doing it, and I didn't want to change the way the ORM is working due to large usage, I did it here in the mutation.

thoughts @jkongie

Reply:
Yes that's correct. Delete should return the object that is being deleted. If the delete operation does not return that object to you, then you will have to fetch it first